### PR TITLE
Fix/instrumentations

### PR DESCRIPTION
**Pull request by heemankv**

### CODE REVIEW

**Review by heemankv — COMMENTED** [review]
Self review done, LGTM!

**Review by ocdbytes — APPROVED** [review]
LGTM !!

File: crates/orchestrator/src/jobs/mod.rs [hunk 1]
```diff
@@ -234,6 +235,10 @@ pub async fn process_job(id: Uuid, config: Arc<Config>) -> Result<(), JobError>
     let external_id = match AssertUnwindSafe(job_handler.process_job(config.clone(), &mut job)).catch_unwind().await {
         Ok(Ok(external_id)) => {
             tracing::debug!(job_id = ?id, "Successfully processed job");
+            // Add the time of processing to the metadata.
+            if job.job_type == JobType::ProofCreation {
```
Comment (apoorvsadana) [inline crates/orchestrator/src/jobs/mod.rs:239]: why only for proof creation? we can add for everything I think

File: crates/orchestrator/src/jobs/mod.rs [hunk 2]
```diff
@@ -341,14 +346,36 @@ pub async fn verify_job(id: Uuid, config: Arc<Config>) -> Result<(), JobError> {
...
+    let attributes = [
+        KeyValue::new("operation_job_type", format!("{:?}", job.job_type)),
+        KeyValue::new("operation_type", "verify_job"),
+        KeyValue::new("operation_verification_status", format!("{:?}", &verification_status)),
+    ];
 
     match verification_status {
         JobVerificationStatus::Verified => {
             tracing::info!(job_id = ?id, "Job verified successfully");
+            if job.job_type == JobType::ProofCreation {
```
Comment (apoorvsadana) [inline crates/orchestrator/src/jobs/mod.rs:360]: same as above

File: crates/orchestrator/src/jobs/mod.rs (hunk 1 above)
Comment (heemankv) [reply crates/orchestrator/src/jobs/mod.rs]: Alright, added.

File: crates/orchestrator/src/jobs/mod.rs (hunk 2 above)
Comment (heemankv) [reply crates/orchestrator/src/jobs/mod.rs]: Alright, added.

**Review by apoorvsadana — APPROVED** [review]

**Pull request merged**